.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#083606",
+        "titleBar.activeBackground": "#0B4B09",
+        "titleBar.activeForeground": "#F2FDF1"
+    }
+}
```

Comment:
I highly recommend deleting this file @EthenThinkful

Reply:
deleted!